environments/sandbox.yml

```
@@ -193,3 +193,10 @@ cname:
   - name: "cdnverify.labs-zcwalkthrough-nodejs"
     ttl: 300
     record: "cdnverify.hmcts-labs-zcwalkthrough-nodejs-shutter-sbox.azureedge.net"
+  - name: "afdverify.labs-louisehuyton-walkthrough"
+    ttl: 300
+    record: "afdverify.hmcts-sbox.azurefd.net"
+    shutter: false
```

this goes on the actual domain one that's missing here:
```suggestion
  - name: "labs-louisehuyton-walkthrough"
    ttl: 300
    record: "hmcts-sbox.azurefd.net"
    shutter: false
```